text/0000-promote-aarch64-unknown-linux-gnu-to-tier1.md

```diff
@@ -0,0 +1,188 @@
+
+ - Arm has dedicated a small team to help improve Rust for the aarch64-unknown-linux-gnu target.
+
+ - **It is very likely that support for aarch64-unknown-linux-gnu in these organisations will trend upwards commensurate with the increasing prevalence of AArch64 silicon based systems.**
+
+Points 3.a through 3.c from the [Guide-level explanation](#Guide-level-explanation) section above are addressed in the [Unresolved questions](#unresolved-questions) section below.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+**There is no drawback envisioned in promoting the Rust aarch64-unknown-linux-gnu to Tier-1.**
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+Given the narrative above, it is the opinion of the author that it would now be tactically sound to promote aarch64-unknown-linux-gnu to Tier-1.
+
+- Inclusion in the Tier-1 category is very likely to be a self sustaining action in that it will promote increased scrutiny with increasing quality as a return. With that return, interest in Rust will grow further both in the AArch64 context and even more generally.
+
+- Anecdotally, not having the Tier-1 'badge' has been seen to become an obstacle to increasing mindshare in Rust for this target. Organisations tend to associate a Tier-1 categorisation with better quality, suitability for key projects, longevity etc. With a reasonably justified Tier-1 badge in place, the likelihood is that such organisations will tend to promote the use of Rust in production.
+
+As such **there is no substantially robust reason to not proceed with promoting aarch64-unknown-linux-gnu to Tier-1.**
+
+# Prior art
+[prior-art]: #prior-art
+
+- Existing Tier-1 targets represent prior-art.
+
+- It is appropriate to call out that no non i686 or x86_64 based target has ever been promoted to Tier-1. The fact that those targets have intrinsically supported self hosted development has arguably been a primary reason for their maturity.
+
+- The aarch64-unknown-linux-gnu target is therefore somewhat uncharted territory.
+
+However, as emphasised in the narrative thus far, **the aarch64-unknown-linux-gnu target now exhibits the properties required by a Tier-1 target as per the target tier policy.**
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+The following points are 'unresolved' at present. From the author's PoV, these are next steps, subject to agreement with the RFC reviewers.
+
+**3.a. An approval from the Compiler Team that Tier-1 target requirements have been met.**
+
+**3.b. An approval from the Infrastructure Team that the target in question may be integrated into CI.**
+
+**3.c. An approval from the Release Team that supporting the target in question is viable in the long term.**
+
+The expectation is that these will get resolved as a result of the discussion that ensues with this RFC posting.
```

Comment:
As a nit, I don't think these need to be "unresolved questions". The approval of those teams is a required next step for this to proceed, but I'd just position them as next steps in the summary, rather than as unresolved questions.

Reply:
Hi @joshtriplett!

Thanks! I completely agree. I did try and explain that I'm treating these more as next steps. The 'Unresolved Questions' sections just seemed to me as a suitable venue for those at the time.

Reply:
I've added a commit that refactors the RFC to address @joshtriplett 's comments.